src/components/TryExtensionLinkDialog.tsx

```diff
@@ -100,11 +100,30 @@ export const TryExtensionLinkDialog: FunctionComponent<{
 const TryExtensionBox: FunctionComponent<{
   onInstallClick: () => void
 }> = props => {
+  const [isClicked, setIsClicked] = useState(false)
+
+  const clickHandler = () => {
+    if (isClicked) {
+      window.location.reload()
+    } else {
+      props.onInstallClick()
```

Comment:
```suggestion
}> = ({ onInstallClick }) => {
  const [isClicked, setIsClicked] = useState(false)

  const clickHandler = () => {
    if (isClicked) {
      window.location.reload()
    } else {
      onInstallClick();
```
otherwise, the `props` object is passed as the receiver to the `onInstallClick` prop, which is not something that should be exposed.